### PR TITLE
Fix Generate Sets Errors

### PR DESCRIPTION
**Pull request by phenomenal-manish**
*Issue #, if available:*

*Description of changes:*
Fix Generate Sets Errors

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by BardiyaAk** [comment]
This seems overlapping with PR #18

File: setlexsem/generate/sample.py [hunk 1]
```diff
@@ -105,6 +105,8 @@ def __init__(self, m_A: int, m_B: int, item_len=None, random_state=None):
         self.m_A = m_A
         self.m_B = m_B
         self.item_len = item_len
+        if isinstance(self.item_len, str):
+            self.item_len = eval(self.item_len)
```
Comment (ndronen) [inline setlexsem/generate/sample.py:109]: Why `eval` instead of `int`?

**Review by ndronen — COMMENTED** [review]
See my question about `eval` vs. `int` for `item_len`.

File: setlexsem/generate/sample.py (hunk 1 above)
Comment (phenomenal-manish) [reply setlexsem/generate/sample.py:109]: When item_len is None (string), we want to convert it to None object, hence, we use eval here.

**Comment by phenomenal-manish** [comment]
> This seems overlapping with PR #18

Yes, I can remove these files from this PR.

File: setlexsem/generate/sample.py (hunk 1 above)
Comment (ndronen) [reply setlexsem/generate/sample.py:109]: Gotcha. Thanks.

**Review by ndronen — APPROVED** [review]
LGTM

**Review by BardiyaAk — APPROVED** [review]

**Pull request merged**